plugins/woocommerce/client/legacy/js/frontend/checkout.js

```diff
@@ -685,14 +685,14 @@ jQuery( function( $ ) {
 				.focus()
 				.addClass( 'has-error' )
 				.attr( 'aria-invalid', 'true' )
-				.attr( 'aria-describedby', 'coupon-error-notice' );
+				.attr( 'aria-describedby', 'coupon-error-notice coupon-code-input-description' );
```

Comment:
If a store has a template override that does not include this selector, this JS will break? This comment applies to the line change below as well.

Reply:
Sorry I missed this was setting the attr.